stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/viewmodel/ChannelsViewModelBinding.kt

```diff
@@ -24,7 +25,11 @@ public fun ChannelsViewModel.bindView(
                 }
             }
             is ChannelsViewModel.State.Result -> {
-                view.setChannels(channelState.channels)
+                channelState
+                    .channels
+                    .map { ChannelListItem.ChannelItem(it) }
```

Comment:
I believe you can use function reference here. `.map(ChannelListItem::ChannelItem)`

Reply:
True 👍